package.json

```diff
@@ -60,6 +60,7 @@
     "react-router": "^2.8.1"
   },
   "dependencies": {
+    "consus-flux": "^0.3.0",
```

Comment:
Sorry, I lied earlier and now we have `consus-flux` as both a dependency and devDependency. Could you please run `npm uninstall consus-flux --save` and then `npm update consus-flux --save-dev`?